ci-operator/config/openshift/osd-network-verifier/openshift-osd-network-verifier-main.yaml

```diff
@@ -0,0 +1,22 @@
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: test
```

Comment:
we may consider adding also `make build` as a condition as it has fmt&mod checks